### DISCUSSION TITLE
[Icon Request] Robot icon

### DISCUSSION THREAD

**Discussion by stefangalescu** [tailwindlabs/heroicons#494] [Ideas]
Reference: https://fontawesome.com/icons/robot

**Comment by hexrw** [comment]
That's a great idea

**Comment by brunogoossens** [comment]
Can be handy when showing system actions

**Comment by ndri** [comment]
I added `robot` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=robot
